frontend/src/Routing/AppRoutes.js

```diff
@@ -2,13 +2,15 @@ import React from 'react'
 import { BrowserRouter, Routes, Route } from 'react-router-dom'
 import Home from '../components/Home/Home'
 import Register from '../components/Register/Register'
+import Resources from '../components/Resources/Resources'
 
 export default function AppRoutes() {
     return (
         <BrowserRouter >
             <Routes>
                 <Route path='/' element={<Home />} />
-                <Route path="/register" element={<Register condition={false} />} />
+                <Route path="/register" element={<Register />} />
```

Comment:
was the condition prop on register eliminated on purpose?

Reply:
It's below. Right under the /resources Route component in this file (Line 14). They only thing I changed was whether the condition was initially false and swapped it to true

Reply:
Will revert changes in this area just in case.